.github/workflows/test-report.yaml

```diff
@@ -0,0 +1,67 @@
+name: Test Report
+
+on:
+  schedule:
+    # Run 2h after the daily tests.yaml
+    - cron: "0 8,20 * * *"
+  workflow_dispatch:
+
+permissions:
+  contents: write
+
+jobs:
+  test-report:
+    name: Test Report
+    # Do not run the report job on forks
+    if: github.repository == 'dask/dask' || github.event_name == 'workflow_dispatch'
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ github.token }}
+      PYTHON_VERSION: 3.9
+    defaults:
+      run:
+        shell: bash -l {0}
+    steps:
+      - uses: actions/checkout@v4.1.1
+        with:
+          repository: dask/distributed
+
+      - name: Setup Conda Environment
+        uses: conda-incubator/setup-miniconda@v3.0.1
+        with:
+          miniforge-variant: Mambaforge
+          miniforge-version: latest
+          condarc-file: continuous_integration/condarc
+          use-mamba: true
+          python-version: ${{ env.PYTHON_VERSION }}
+          environment-file: continuous_integration/scripts/test-report-environment.yml
+          activate-environment: dask
+
+      - name: Show conda options
+        run: conda config --show
+
+      - name: mamba list
+        run: mamba list
+
+      - uses: actions/cache@v3
+        id: cache
+        with:
+          # Suffix is depending on the backend / OS. Let's be agnostic here
+          # See https://docs.python.org/3/library/shelve.html#shelve.open
+          path: |
+            test_report*
+            !test_report.html
+          key: ${{ env.PYTHON_VERSION }}-${{ hashFiles('continuous_integration/scripts/test_report*') }}
```

Comment:
```suggestion
                    # Note: these files are in the dask/distributed repository
                    key: ${{ hashFiles('continuous_integration/scripts/test_report*') }}
```

Reply:
Do I understand correctly that the cache is specific to dask/dask, so we don't need to match the key in dask/distributed?

Reply:
That's my understanding as well: 
https://github.com/milesgranger/dask/actions/runs/7524967710/job/20480569721#step:14:2